cvat-ui/src/reducers/tasks-reducer.ts

```diff
@@ -72,6 +73,15 @@ export default (state: TasksState = defaultState, action: AnyAction): TasksState
                 gettingQuery: { ...action.payload.query },
             };
         }
+
+        case TasksActionTypes.UPDATE_CURRENT_JOBS_PAGE: {
+            return {
+                ...state,
+                current: state.current.map((current, i) =>
+                    (i === 0 ? { ...current, currentJobsPage: action.payload.pageNumber } : current)),
```

Comment:
Probably ``i===0`` is not correct condition. We should use ``taskID`` from payload.